cmd/scanner/scanner.go

```diff
@@ -71,46 +91,60 @@ func getCloudProviderAccounts() ([]inventory.Account, error) {
 			account.Key("user").String(),
 			account.Key("key").String(),
 		)
-		accounts = append(accounts, *newAccount)
+		if err := s.inventory.AddAccount(*newAccount); err != nil {
+			return err
+		}
 	}
 
-	return accounts, nil
+	return nil
 }
 
 // createStockers creates and configures stocker instances for each provided account to be inventoried.
-func createStockers(accounts []inventory.Account) error {
-	for _, account := range accounts {
+func (s *Scanner) createStockers() error {
+	for i := range s.inventory.Accounts {
+		account := s.inventory.Accounts[i]
 		switch account.Provider {
 		case inventory.AWSProvider:
-			logger.Info("Adding the AWS account to be inventoried", zap.String("account", account.Name))
-			stockers = append(stockers, stocker.NewAWSStocker(account))
+			s.logger.Info("Adding the AWS account to be inventoried", zap.String("account", account.Name))
+			s.stockers = append(s.stockers, stocker.NewAWSStocker(&account, logger))
 		case inventory.GCPProvider:
-			logger.Warn("Failed to scan GCP account",
+			logger.Warn("Can't scan GCP account",
 				zap.String("account", account.Name),
 				zap.String("reason", "not implemented"),
 			)
+			// TODO: Uncomment line below when Stocker is implemented
+			//s.stockers = append(s.stockers, stocker.NewGCPStocker(&account, logger))
 		case inventory.AzureProvider:
-			logger.Warn("Failed to scan Azure account",
+			logger.Warn("Can't scan Azure account",
 				zap.String("account", account.Name),
 				zap.String("reason", "not implemented"),
 			)
+			// TODO: Uncomment line below when Stocker is implemented
+			//s.stockers = append(s.stockers, stocker.NewAzureStocker(&account, logger))
 		}
 	}
 
-	if len(stockers) == 0 {
-		return fmt.Errorf("no valid stockers created")
+	if len(s.stockers) == 0 {
+		s.logger.Warn("No accounts has been provided for scan", zap.String("credentials_file", s.credsFile))
+	}
+
+	if s.logger.Core().Enabled(zap.DebugLevel) {
```

Comment:
I don't understand how it works.
Should I configure any env vars?
Give me an example please

Reply:
The env var to enable/disable Debug mode is called. "CIQ_LOG_LEVEL", and it's being parsed on the "NewLogger" function at 'internal/logger/logger.go' file. 

Each time we instance a new 'internal/logger' object, it checks the value of that envVar to configure the logger accordingly